integration/bin/tachyon-mesos.sh

```diff
@@ -7,13 +7,52 @@ SCRIPT_DIR="$(cd "$(dirname "$0")"; pwd)"
 source "${SCRIPT_DIR}/common.sh"
```

Comment:
Please adhere to the Google shell style guide: https://google.github.io/styleguide/shell.xml